src/components/course/styles/_CourseHeader.scss

```diff
@@ -1,3 +1,11 @@
 .course-header {
   box-shadow: $box-shadow-sm;
 }
+
+.course-badge {
```

Comment:
suggested: perhaps a more specific selector here to indicate which component we are in ? esp if you refactor to its own component, then it makes sense to call it something like `.course-skills .course-badge` ?